cms/static/js/views/pages/paged_container.js

```diff
@@ -21,16 +24,30 @@ define(["jquery", "underscore", "gettext", "js/views/pages/container", "js/views
                });
             },
 
-            refreshXBlock: function(element, block_added) {
+            refreshXBlock: function(element, block_added, is_duplicate) {
                 var xblockElement = this.findXBlockElement(element),
                     rootLocator = this.xblockView.model.id;
                 if (xblockElement.length === 0 || xblockElement.data('locator') === rootLocator) {
                     this.render({refresh: true, block_added: block_added});
                 } else {
-                    this.refreshChildXBlock(xblockElement, block_added);
+                    this.refreshChildXBlock(xblockElement, block_added, is_duplicate);
                 }
-            }
+            },
+
+            toggleChildrenPreviews: function(xblockElement) {
+                xblockElement.preventDefault();
+                this.xblockView.togglePreviews();
+            },
+
+            updatePreviewButton: function(show_previews){
+                var text = (show_previews) ? gettext('Hide Previews') : gettext('Show Previews'),
+                    icon_class = (show_previews) ? 'fa-arrow-up' : 'fa-arrow-down',
```

Comment:
I should remove this code anyway, since I believe I was asked to use the recycle code in the last review, so not only does this code do nothing, it should also not be made to work. Doing so...